.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
do not push files from .idea to directory, add in file BooksReviewPlatform/client/.gitignore .idea/